src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
+  const menuItems = [
+    {
+      html: "Logout",
+      id: "logout",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  var userLabel = AuthStore.getUserLabel();
+  return (
+    <AccountDropdown
+      menuItems={menuItems}
+      userName={userLabel}
+      willAnchorRight={false}
+    />
+  );
+}
+
+function getClusterDropdown() {
+  const menuItems = [
+    {
+      html: "Install CLI",
+      id: "install-cli",
+      onClick: AuthStore.logout
```

Comment:
This seems odd...

Reply:
Yeah, my intention was it to be a placeholder for you to fill in.